src/test/jtx/Env.h

```diff
@@ -326,7 +326,7 @@ class Env
             The Application network time is set to
             the close time of the resulting ledger.
     */
-    void
+    bool
```

Comment:
Needs a brief comment explaining what the return value does (true, no error; false error).

Reply:
Done